### ISSUE TITLE
the model iterate too fast and the MAP doesn't change during the iteration

### ISSUE THREAD

**Issue by InfiniteShen** [faneshion/MatchZoo#68]
I have tried to run the model on wikiQA corpus. I  first run the run_data.sh and later I tried some 
of the models using the given 'python matchzoo/main.py --phase train --model_file 
examples/wikiqa/config/drmm_wikiqa.config'. However during the 400 iterations the MAP and the loss 
didn't rise and it run so fast. The 400 iterations took only 2 minites and the MAP is only about 30%.Did
 anyone know why this would happen? Thanks

**Comment by zhouzhouyang520** [comment]
If your learning rate is too small, it would be happen.Or are you changed the parameter of config data like ‘query_per_iter’‘batch_per_iter’? Could you show me your config data of drmm?

**Comment by InfiniteShen** [comment]
Thank you so much ! It's exactly the problem that the learning rate is too low. I thought the parameters in the config was already settled so I didn't change the learning rate.
